utils/errors.ts

```diff
@@ -1,9 +1,18 @@
+import * as Sentry from "@sentry/nextjs";
+import { ContentfulSynth } from "./contentful";
+
 export function errorFilter(value: unknown | Error): boolean {
   if (value instanceof Error) {
-    // TODO: Have a proper error reporting mechanism
-    console.log(`Errored at`);
-    console.log(value);
+    Sentry.captureException(value);
     return false;
   }
   return true;
 }
+
+export class SynthFetchingError extends Error {
+  public constructor(message: string, synth: ContentfulSynth) {
```

Comment:
why not also save the stack trace here?  and is this is just a custom error that doesnt have to do with sentry?

Reply:
No reason really, I didn't think of adding it and Sentry adds this by default. The custom error class has nothing to do with Sentry per se, its just that the error message for this exception is `Failed to fetch, 500 error` and I wanted to include in the error which synth caused the error

Reply:
got it